modules/sdk-core/src/account-lib/baseCoin/enum.ts

```diff
@@ -41,6 +41,10 @@ export enum TransactionType {
   Batch,
   // Claim the staking rewards
   StakingClaim,
+  // Add delegator to p-chain
+  addDelegator,
+  // Add validator to p-chain
+  addValidator,
```

Comment:
```suggestion
  // Add delegator to p-chain
  AddDelegator,
  // Add validator to p-chain
  AddValidator,
```

Reply:
What is it the impact of new transaction type?

Could StakingLock be the transaction type Instead of increase transaction types and add unnecessary complexion?